### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -89,7 +89,7 @@
         "symfony/css-selector": "^4.4 || ^5.3 || ^6.0",
         "symfony/filesystem": "^4.4 || ^5.3 || ^6.0",
         "symfony/maker-bundle": "^1.25",
-        "symfony/phpunit-bridge": "^5.3 || ^6.0",
+        "symfony/phpunit-bridge": "^6.0",
```
Comment (jordisala1991) [inline composer.json:92]: I think there are some new versions of symfony contracts, can you check if composer outdated is clean?

Comment (VincentLanglet) [reply composer.json:92]: Done

File: composer.json [hunk 2]
```diff
@@ -60,7 +60,7 @@
         "symfony/security-csrf": "^4.4 || ^5.3 || ^6.0",
         "symfony/string": "^5.3 || ^6.0",
         "symfony/translation": "^4.4 || ^5.3 || ^6.0",
-        "symfony/translation-contracts": "^1.1.10 || ^2.3",
+        "symfony/translation-contracts": "^1.1.10 || ^2.3 || ^3.0",
```
Comment (jordisala1991) [inline composer.json:63]: Which package is blocking the upgrade to 3? it is not getting installed.

Comment (VincentLanglet) [reply composer.json:63]: twig/string-extra

Comment (jordisala1991) [reply composer.json:63]: What should we do? revert this one and wait for string extra? Not sure what changes for this specific contract

Comment (VincentLanglet) [reply composer.json:63]: Looking at the changelog, https://github.com/symfony/contracts/blob/main/CHANGELOG.md#30, I would consider ok to keep it like this. As u want

**Review by jordisala1991 — APPROVED** [review]

**Pull request merged**